.github/actions/linux/action.yml

```diff
@@ -0,0 +1,19 @@
+name: "Run system tests"
+description: "Run system tests for the specified module"
+inputs:
+  module_name:
+    description: "module name"
+    required: true
+    default: ""
+
+runs:
+  using: "composite"
+  steps:
+    - run: |
+        source /etc/bashrc
+        cd generated
+        mkdir junit
+        mkdir kibana
+        cd ${{ inputs.module_name }}
+        python -m tox -c tox-system_tests.ini
+      shell: bash
```

Comment:
this file is identical to the windows one except for the shell.

Could you have one file and make that an input like module_name?
Could you then move the file up one directory and rename to run_system_tests_action.yml or something like that?

Reply:
Could you have one file and make that an input like module_name?
It's a good point but I could not combine in single file, github action composites does not have flexibity unlike azure pipelines.

Could you then move the file up one directory and rename to run_system_tests_action.yml or something like that?
github composite action only understands action.yml so we cannot rename to run_system_tests_action.